### ISSUE TITLE
Fatal: UnexpectedValueException

### ISSUE THREAD

**Issue by LokeYourC3PH** [pondersource/mfazones#7]
Just going to drop this here. Happened after installing the app. File Access Control is also installed fyi. It spams the log every second, so I had to disable it for now.

CPU: ARMx64 Architecture
OS: Ubuntu 22.04
NextCloud: 25.0.5

```
[no app in context] Fatal: UnexpectedValueException: Check OCA\FilesAccessControl\Operation does not exist at <<closure>>

 0. /home/sysadmin/web/**REDACTED**/public_html/apps/workflowengine/lib/Manager.php line 336
    OCA\WorkflowEngine\Manager->validateOperation("OCA\\FilesAccessControl\\Operation", "", [["OCA\\Workflow ... ]], "deny", ["OCA\\WorkflowE ... "], "OCA\\WorkflowEngine\\Entity\\File", [])
 1. /home/sysadmin/web/**REDACTED**/public_html/apps/mfazones/lib/AppInfo/Application.php line 129
    OCA\WorkflowEngine\Manager->addOperation("OCA\\FilesAccessControl\\Operation", "", [["OCA\\Workflow ... ]], "deny", ["OCA\\WorkflowE ... "], "OCA\\WorkflowEngine\\Entity\\File", [])
 2. /home/sysadmin/web/**REDACTED**/public_html/apps/mfazones/lib/AppInfo/Application.php line 60
    OCA\mfazones\AppInfo\Application->addFlows()
 3. /home/sysadmin/web/**REDACTED**/public_html/lib/private/ServerContainer.php line 107
    OCA\mfazones\AppInfo\Application->__construct()
 4. /home/sysadmin/web/**REDACTED**/public_html/lib/private/ServerContainer.php line 176
    OC\ServerContainer->getAppContainer("mfazones", "mfazones")
 5. /home/sysadmin/web/**REDACTED**/public_html/lib/private/ServerContainer.php line 143
    OC\ServerContainer->getAppContainerForService("OCA\\mfazones\\AppInfo\\Application")
 6. /home/sysadmin/web/**REDACTED**/public_html/lib/private/AppFramework/Bootstrap/Coordinator.php line 186
    OC\ServerContainer->query("OCA\\mfazones\\AppInfo\\Application")
 7. /home/sysadmin/web/**REDACTED**/public_html/lib/private/legacy/OC_App.php line 208
    OC\AppFramework\Bootstrap\Coordinator->bootApp("mfazones")
 8. /home/sysadmin/web/**REDACTED**/public_html/lib/private/legacy/OC_App.php line 141
    OC_App::loadApp("mfazones")
 9. /home/sysadmin/web/**REDACTED**/public_html/lib/base.php line 1027
    OC_App::loadApps()
10. /home/sysadmin/web/**REDACTED**/public_html/index.php line 36
    OC::handleRequest()

POST /settings/apps/disable
from xxx.xxx.xxx.xxx by sysadmin at 2023-03-28T13:28:54+00:00
```

**Comment by navid-shokri** [comment]
@mrvahedi68

**Comment by michielbdejong** [comment]
Ah, sorry you ran into this! We have a more complete documentation of the context of this app that we had forgotten to link in the readme of this repo. Fixed in https://github.com/pondersource/mfazones/commit/16ef3c472b75afeb77752e8351bea131188a4a74 now.

We developed this app for Swedish universities who run Nextcloud and want to apply the MFA requirement for specific folders ("MFA Zones") but at the same time benefit from SSO through SAML and Global Site Selector.

But we also want this app to become generally available for anyone to use, so we are very interested to hear how you will be using this app, and welcome your feedback, thanks for this first issue!

If you want we can help you patch your server so that the "MFA Verified" Check becomes available in the Work Flow Engine, it will also help us learn more about how to make our app generally usable.

Sorry again that we hadn't documented this issue properly so far.
